server/config/routes.js

```diff
@@ -25,6 +25,7 @@ var inventory            = require('../controllers/stock/inventory');
 var depots               = require('../controllers/stock/depot');
 var consumptionLoss      = require('../controllers/stock/inventory/depreciate/consumptionLoss');
 var trialbalance         = require('../controllers/finance/trialbalance');
+var journal              = require('../controllers/finance/journal/index');
```

Comment:
For future reference, when the following are equivalent:
```js
var journal = require('../controllers/finance/journal/index'):
// ... is the same as ...
var journal = require('../controllers/finance/journal');
```
This is because of the way NodeJS loads modules.  [Here is the documentation](https://nodejs.org/api/modules.html#modules_folders_as_modules).

From the documentation:
> If there is no package.json file present in the directory, then Node.js will attempt to load an index.js or index.node file out of that directory. For example, if there was no package.json file in the above example, then require('./some-library') would attempt to load:
 - ./some-library/index.js
 - ./some-library/index.node